python/tests/core/test_declarative_schema.py

```diff
@@ -202,3 +203,29 @@ class UnknownType:
             reference_metrics = set(reference_results.get_column(column).get_metric_names())
             declarative_metrics = set(declarative_results.get_column(column).get_metric_names())
             assert reference_metrics == declarative_metrics
+
+
+def test_augmenters_row() -> None:
+    schema = DeclarativeSchema(
+        STANDARD_RESOLVER,
+        augmenters=[AugmenterSpec(column_name="col1", callbacks={"col2": lambda x: x, "col3": lambda x: x})],
```

Comment:
what happens if the callback specify the same col name (col1)? will it overwrite the original column?
what would be the ideal behavior in this case? let it overwrite or create new cols, like `col1_aug_1` or something similar?

Reply:
Good question :)

Reply:
Current implementation will overwrite the original input. We should at least emit an INFO log message that the derived column name exists in the input already.